docs/product-assembly_case-management-getting-started/src/main/asciidoc/case-management-designing-a-case-proc.adoc

```diff
@@ -1,2 +1,87 @@
++
+For more information about the case file, see <<case-management-case-file-con-{context}>>.
+
+. *Define the roles involved in the case.*
++
+Open the properties panel and click the down arrow next to `Variable Definitions` to open the `Editor for Variable Definitions`.For more information about roles, see <<case-management-roles-con-{context}>>.
+
+. *Define what work item will be triggered when a new case starts.*
++
+As case definition is an ad hoc process definition, it does not require an explicit start node. Nodes that have no incoming connections and are marked as `Adhoc Autostart` and are automatically triggered when an instance is started.
++
+By default, the case definition designer includes a `Milestone` object configured as `Adhoc autostart` in the palette. Modify this object or replace it with another work item, such as a user task or script task to be triggered when a new case is started.
++
+In the IT Orders example, the `Prepare hardware spec` user task and the `Milestone 1: Order placed` milestones are `Adhoc autostart`, and are automatically triggered when an instance is started by a request being made for new IT hardware.
+
+. *Define the `Place order` subprocess.*
++
+In the IT Orders example, the `place-order` process is a separate business process that is carried out by the supplier. This is a reusable process that occurs during the course of case execution. 
+.. From the project menu, click *Add Asset* -> *Business Process* to create a new business process. Define the name and package for the new process and click *Ok*. 
+.. Define the process that is a subprocess within the case. For example, the IT Orders `Place order` process is a simple user task that applies to the supplier group and receives the relevant order data and case variables directly from the case file.
+
+. *Define milestones*
++
+Add a milestone to the case definition by selecting `Milestone` from the *Object Library* and dragging it on to the case designer. The following milestones are configured in the IT Orders sample project:
++
+* `Milestone 1: Order placed` is `Adhoc autostart`. This milestone is in a wait state until the condition is met. The condition for this milestone is that a case file variable named `ordered` exists with the value `true`. A script task then sends a notification that the hardware order has been placed.
+* `Milestone 2: Order shipped` is triggered by a signal event once the order is ready to be sent. A script task sends a notification once the order is shipped  and added to the tracking system.
+* `Milestone 3: Delivered to customer` is triggered by a signal event once the order has successfully shipped to the customer. Once this milestone is reached, a `Customer satisfaction survey` is sent. 
```

Comment:
Same as Milestone 1 for completion but checking variable name _delivered_. Once completed then _Create customer survey_ task is created.

Reply:
Done. These milestones have been moved from the procedure to the _case-management-itorders-milestones-proc_ file, which will be updated further as part of the work for the Milestones JIRA.